src/Client.php

```diff
@@ -25,56 +25,38 @@ public function create(Sandbox $sandbox): Sandbox
     {
         $jobData = \GuzzleHttp\json_encode($sandbox->toApiRequest());
         $request = new Request('POST', 'sandboxes', [], $jobData);
-        try {
```

Comment:
I think these try/catchs are redundant or do you think not?

Reply:
yes, I think they are!